src/schema/v2/me/__tests__/similarToRecentlyViewed.test.ts

```diff
@@ -99,4 +100,61 @@ describe("SimilarToRecentlyViewed", () => {
     })
     expect.assertions(1)
   })
+
+  it("when using an impersonated call", async () => {
+    const query = gql`
+      {
+        me {
+          similarToRecentlyViewedConnection(first: 2) {
+            edges {
+              node {
+                slug
+                title
+              }
+            }
+            pageInfo {
+              hasNextPage
+            }
+          }
+        }
+      }
+    `
+    const artworks = [
+      { id: "x", title: "X the snail" },
+      { id: "y", title: "Y the Person" },
+      { id: "z", title: "Z the butterfly" },
+    ]
+
+    const ids = ["x", "y", "z"]
+
+    context.meLoader = async () => jest.fn().mockResolvedValue(null)
```

Comment:
Is this right? I think an impersonated call _will_ have `meLoader` available, but the call to `meLoader` will then fail (vs. returning `null`) as the `/me` endpoint isn't enabled for impersonation.

Reply:
for some reason works both ways, but I will remove it from here.